spirit_utils/src/rviz_interface.cpp

```diff
@@ -123,6 +127,29 @@ void RVizInterface::footstepPlanCallback(const spirit_msgs::FootstepPlan::ConstP
   footstep_plan_viz_pub_.publish(points);
 }
 
+void RVizInterface::stateEstimateCallback(const spirit_msgs::StateEstimate::ConstPtr& msg) {
+
+  // Make a transform message for the body, populate with state estimate data, and publish
+  geometry_msgs::TransformStamped transformStamped;
+  transformStamped.header = msg->header;
+  transformStamped.header.frame_id = map_frame_;
+  transformStamped.child_frame_id = "dummy";
+  transformStamped.transform.translation.x = msg->body.pose.pose.position.x;
+  transformStamped.transform.translation.y = msg->body.pose.pose.position.y;
+  transformStamped.transform.translation.z = msg->body.pose.pose.position.z;
```

Comment:
Irritating that geometry_msgs::Transform and geometry_msgs::Pose use different representations of a 3D point, not worth doing anything about it though.